### PR TITLE
re-format the Rates table

### PR DESCRIPTION
**Pull request by afeld**
This pull request does a few things to the Access Package section of the Rates page:
- Flips the axes of the table
  - It feels more natural to me this way, though I recognize that pricing options (especially with checkboxes for features) are sometimes laid out horizontally.
- Adds a `Usage fee?` column, to split that from the base price
- Converts it from an HTML table to a Markdown one
  - Since I wasn't able to keep the `class` on the `<table>`, this inadvertently caused a styling change. This wasn't my intention, though it does it make it consistent with the other tables on the site.
- Minor tweaks to the verbiage

**Before**

Rates table screenshot before the change

**After**

Rates table screenshot after the change

### CODE REVIEW

File: content/intro/pricing/rates.md [hunk 1]
```diff
@@ -18,44 +18,22 @@ Find out [more about this pricing model]({{< relref "intro/pricing/pricing-model
...
-    <td>$15K / year plus usage charges</th>
-    <td>$20K / year plus usage charges</th>
-    <td>$90K / year plus usage charges</th>
-  </tr>
-</table>
-
-Access package fees are charged yearly and are non-severable. 
+| Package | What's included? | Base annual fee | Usage charges? |
+| --- | --- | --- | --- |
+| **Sandbox** | Any user with a `.gov` or `.mil` email address can try out cloud.gov, no paperwork required. All managed services are available. Resource usage is capped, though it can be raised. | Free | No |
```
Comment (afeld) [inline content/intro/pricing/rates.md:23]: "Sandbox usage caps can be raised" used to be listed under the `Prototyping` package...not sure if that was intentional?

**Comment by afeld** [comment]
Updated to clarify that, per @mogul [in Slack](https://18f.slack.com/archives/cloud-gov/p1470784519000802), we aren't currently charging for managed services.

**Comment by brittag** [comment]
I looked at this locally, and these changes look good to me. I find the reoriented table easier to read - good catch.

@berndverst do you want to also take a look at this and merge it if it looks good to you?

**Comment by berndverst** [comment]
Looks great. Merging.

**Pull request merged**